lib/js/components/Dropdown/Dropdown.stories.mdx

```diff
@@ -79,7 +79,7 @@ import { ICONS } from '../Icon';
 			template:
 				'<div style="position: relative">' +
 				'<dropdown :trigger-action="triggerAction" :force-show="forceShow" :same-width="sameWidth">' +
-				'<span slot="reference">Dropdown entry point</span>' +
+				'<template #reference><span>Dropdown entry point</span></template>' +
```

Comment:
It breaks IDE for me:
<img width="686" alt="Screenshot 2022-11-02 at 08 37 15" src="https://user-images.githubusercontent.com/7030884/199427927-aedfab16-5dd3-40c3-9223-34a195c81e22.png">

Same as https://github.com/bethinkpl/wnl-platform/pull/3733#issuecomment-1290097763

Reply:
Solved by migrating to TS [`a1ed77b` (#219)](https://github.com/bethinkpl/design-system/pull/219/commits/a1ed77b30d3c307a0957602ff296cf6bd907094e)